src/js/utils.js

```diff
@@ -107,8 +107,13 @@ this.exports = {
   },
   
   hasSensors: function(thermostat) {
-    return true; //TODO: Fixme!
+    var hasSensors = false;
+    thermostat.remoteSensors.forEach(
+      function(sensor) {
+          if (sensor.type ==='ecobee3_remote_sensor') {
```

Comment:
This might need some improvement. I have a third-party sensor that I ordered from flair.co and I'm not sure what this is going to say here. Maybe it should just be !=='thermostat'.